File: quines/quine.py
stanza = """

def handler(body, ctx):
    marker = chr(34) * 3
    quine = 'stanza = ' + marker + stanza + marker + stanza
    return ctx.HTTPResponse(body='{}'.format(quine))
"""

def handler(body, ctx):
    marker = chr(34) * 3
    quine = 'stanza = ' + marker + stanza + marker + stanza
    return ctx.HTTPResponse(body='{}'.format(quine))

File: quines/quine.js
const stanza = `

exports.handler = async (body, ctx) => {
  const backtick = String.fromCharCode(96);
  const quine = 'const stanza = '
    + backtick + stanza + backtick + ';'
    + stanza;
  return new ctx.Response({
    body: quine,
  });
};
`;

exports.handler = async (body, ctx) => {
  const backtick = String.fromCharCode(96);
  const quine = 'const stanza = '
    + backtick + stanza + backtick + ';'
    + stanza;
  return new ctx.Response({
    body: quine,
  });
};
